quickstarts/kind.md

```diff
@@ -55,9 +55,7 @@ kubectl create namespace kafka
 Next we apply the Strimzi install files, including `ClusterRoles`, `ClusterRoleBindings` and some **Custom Resource Definitions** (`CRDs`). The CRDs define the schemas used for declarative management of the Kafka cluster, Kafka topics and users.
 
 ```shell
-curl -L https://github.com/strimzi/strimzi-kafka-operator/releases/download/{{site.data.releases.operator[0].version}}/strimzi-cluster-operator-{{site.data.releases.operator[0].version}}.yaml \
-  | sed 's/namespace: .*/namespace: kafka/' \
-  | kubectl apply -f - -n kafka 
+kubectl apply -f https://strimzi.io/install/latest?namespace=kafka -n kafka
```

Comment:
Should we quote those?

Reply:
Not sure. Does it happen automatically for you when you copy paste them into terminal? It does for me, but not sure how much this is common future or some MacOS trick.

Reply:
It doesn't get quoted in my (fariyl bog standard) Fedora setup. Presumably there'll be some terminal emulators which do support it. Thinking more, I guess the chances of the shell expanding the `?` are actually pretty close to zero, since it's so unlikely there'll be matching filenames. So I suppose we can leave it like this and fix it if anyone ever complains.

Reply:
So, just to make sure the stuff works weel for you ... would this work fine for you?

```
kubectl apply -f https://strimzi.io/install/latest\?namespace\=kafka
```

Reply:
Yes. Single quotes works too. And just plain`?` works if the shell can't expand the wildcard.